<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>About Myself</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/darkly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="site_libs/anchor-sections-1.0/anchor-sections.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Stephanie Call's Website</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="labs.html">EvoGenLab</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">About Myself</h1>

</div>


<div id="undergraduate-and-graduate-education" class="section level1">
<h1>Undergraduate and Graduate Education</h1>
<p>In 2018, I graduated <em>summa cum laude</em> from the University of Tulsa with a Bachelor’s of Science in Chemical Engineering with a Pre-Medicine Emphasis. After the summer, I moved from Oklahoma to Massachusetts for graduate school at the University of Massachusetts Amherst for a PhD in Chemical Engineering. Shortly thereafter, I joined Lauren Andrew’s Lab, which focuses on synthetic biology and genetic engineering, especially through high-throughput methods.</p>
</div>
<div id="research" class="section level1">
<h1>Research</h1>
<p>My project focuses on the elucidation of gene associated with cell attachment, biofilm formation, and antibiotic resistance in both <em>E. coli</em> and <em>S. aureus</em>. <em>E. coli</em> is the most common cause of catheter-associated urinary tract infections, which account for the greatest number of hospital-acquired infections, while <em>S. aureus</em> is the most common cause of extremely dangerous catheter-associated bloodstream infections. These biofilm infections are extremely resilient and difficult to treat, due in part to greatly increased antibiotic resistance. Additionally, the broad-spectrum antibiotics commonly used to try to treat these biofilm infections are contributing to the development of antibiotic-resistant microbes in hospitals and the community. My aim is to find genes associated with cell attachment and biofilm formation on relevant biomaterials and genes associated with increased antibiotic resistance of these attached bacteria to relevant antibiotics. These genes may provide effective genetic targets for the design of targeted antimicrobials, such as peptide nucleic acids, that could specifically target just pathogenic bacteria to prevent and treat biofilm infections while decreasing the chance of the development of antibiotic resistance in the general microbial population.</p>
<p>To elucidate these genes, I am using CRISPR interference, or CRISPRi, to repress all native genes in the genomes of <em>E. coli</em> MG1655 and <em>S. aureus</em> ATCC 12600. I have been characterizing the dCas9 CRISPRi tool I created for MG1655 to determine the design rules needed to effectively repress gene expression. For ATCC 12600, I have been building and characterizing a genetic part library for <em>S. aureus</em> because the literature reports very few synthetic genetic parts for this organism, with no compiled “toolbox” of various genetic parts all characterized in a standard way. After creating this genetic parts library, I can then build and characterize a dCas12a CRISPRi tool for ATCC 12600 as I did for MG1655.</p>
<div class="figure">
<img src="images/dCas9Binding.png" alt="" />
<p class="caption">CRISPRi is a simple way to specifically target and strongly repress gene expression. The only required components to target a gene is a Protospacer Adjacent Motif (PAM) adjacent to the target site, homology with the 20 nucleotide target sequence, and expression of the single guide RNA (sgRNA) and dCas protein. Additionally, care must be taken to prevent off-target effects and overexpression of the dCas protein. This is easily done through the efficient design of the sgRNA and characterization of the CRISPRi tool, respectively.</p>
</div>
<p>Soon, I will design genome-wide guide RNA pools to systematically target nearly every gene, ideally with at least 10 designs per gene, using the design rules for each CRISPR system. I will then use a pooled competitive attachment assay to measure how the repression of certain genes affects the attachment of bacteria to a range of biomaterials of different compositions, hydrophilicities, and stiffnesses, as these factors have been shown to affect bacterial attachment and biofilm formation. These biomaterials will be created and characterized by our collaborators at the Schiffman Lab here at the University.</p>
<div class="figure">
<img src="images/CRISPRi_overview.png" alt="" />
<p class="caption">Overview of the attachment assay experiment to uncover cell attachment and biofilm formation genes. The guide RNAs will be designed to minimize off-target effects and maximize repression. The library will be synthesized as an oligo pool and then assembled and inserted into the bacteira using high efficiency methods to minimize loss of designs. The attachment assay will be performed over varying lengths of time to study both initial cell attachment and mature biofilm formation. Next generation sequencing will be used to determine the frequency of each sgRNA design before and after the attachment assay, which will be used to determine the genes associated with biofilm formation via the absolute change in frequency.</p>
</div>
<p>After determining the top genes most associated with promoting and inhibiting cell attachment and biofilm formation through the competitive attachment assay, I will investigate the binary genetic interactions (epistasis) between a subset of these genes by inhibiting two genes at once and performing the same attachment assay. The combinations that inhibit cell attachment and/or biofilm formation the most could be effective targets for the design of targeted antimicrobials.</p>
<p>After investigating cell attachment and biofilm formation genes, I will be performing an antibiotic attachment assay using clinically-relevant antibiotics to determine the genes most associated with antibiotic resistance of attached bacteria. Like for the biofilm formation genes, I will then investigate the genetic interactions between the a subset of genes associated with antibiotic resistance as I will with the cell attachment and biofilm formation genes.</p>
<div class="figure">
<img src="images/CRISPRi_overview_epistasis.png" alt="" />
<p class="caption">Overview of the combinatorial attachment assay with antibiotics to determine antibiotic resistance genes. The procedure for this assay is the same as before except that the plasmid library will be assembled with two sgRNA designs for simultaneous repression of two genes. In the case of the antibiotic combinatorial attachment assay, a clinically-relevant antibiotic is present in the media above the biomaterial. The attachment assay to determine cell attachment and biofilm formation genes is the same except without the antibiotic present in the media.</p>
</div>
</div>
<div id="research-interests" class="section level1">
<h1>Research Interests</h1>
<p>My research interests include synthetic biology, genetic engineering, mathematical modeling, bioinformatics, chemistry, and kinetics. I find it incredible and amazing that life exists and is controlled by so many chemical reactions happening simultaneously within a cell and how all the information needed to run and control these chemical reactions are encoded within DNA. I also like how we can harness these cells to produce non-native products or change their behavior/function by simply introducing additional DNA in the form of designed synthetic genes. I can use bioinformatics to, for example, investigate how the genome is affected by introduction of a foreign gene or search for patterns in gene expression under different conditions. (I also really like programming).</p>
<p>On a related note, I enjoy mathematical modeling of both life processes (e.g., the binding and unbinding of transcription factors to DNA or the prediction of the translation initiation rate of a gene based on the 5’ untranslated region sequence) and non-life processes (e.g., reaction kinetics of a chemical process or the dynamic response of an electrical circuit to the introduction of a pulsed voltage). Preferably, these models would be derived from first principles, such as the Laws of Thermodynamics or statistical mechanics, not simply black-box models. It’s incredible that we can use numbers and mathematics, both arbitrary human constructs, to accurately predict the outcome of a situation based on known parameters.</p>
<p>Finally, I also enjoy chemistry and kinetics. I am amazed how a new matter with different and new properties can be made from other matter through chemical reactions. In conjunction with mathematical modeling, I enjoy investigating the kinetics of these chemical reactions and trying to figure out what is happening on the molecular level when these reactions occur, such as the movement of electrons, molecular mechanisms, and more.</p>
</div>
<div id="other-interests" class="section level1">
<h1>Other Interests</h1>
<p>Besides conducting research, I have many other interests and hobbies. The most prominent is my love of rowing and working out in general. With no prior experience on a competitive sports team, I joined the University of Tulsa’s Women’s Rowing Team my freshman year of undergraduate school and soon developed a strong passion for the sport and physical activity. My boat even won a gold medal at the Conference Championships my Sophomore year. I have continued a strict training regimen since graduating from the University of Tulsa, and I joined a local competitive rowing club in Northampton and continue to race at local and regional regattas. Although I don’t row as much due to the time, money, and weather constraints, I instead cycle indoors on a magnetic spin bike I bought for personal use or run when I can. I hope to one day complete a full Ironman within the top 20% of finishers.</p>
<div class="figure">
<img src="images/Rowing.JPG" alt="" />
<p class="caption">I am rowing a single shell, my favorite type of boat, on the Connecticut River with the Northampton Community Rowing Club. I have even won a couple of medals since joining the Rowing Club, including a bronze in the racing single above at the 2019 Head of the Riverfront Regatta.</p>
</div>
<p>My other hobbies and interests include video games, programming, and crocheting. Some of my favorite video game franchises include Minecraft, the original Spyro trilogy, Pokemon, and Assassin’s Creed.</p>
<div class="figure">
<img src="images/Minecraft_promo.jpg" alt="" />
<p class="caption">Minecraft - like Lego on a computer but much, much more.</p>
</div>
<p>I recently gained an interest in programming after teaching myself Python over the first few months of quarantine at the beginning of this global pandemic. I enjoy how I can customize and create my own code, both for work and to modify video games like Minecraft. It’s especially satisfying to create a script that automates data analysis, like my flow cytometry data. I have even created custom scripts for my lab.</p>
<p><a href="images/python_r.jpg">I currently know how to program in Python and R, and I’ve been using these languages to make scripts. I also know a little about Java because Minecraft is written in Java and it is easy (and common) to modify the game, including through a command line in-game.</a></p>
<p>When I have the time, I also love to crochet. I like to make scarves and hats with a variety of different stitches and patterns for warmth in the wintertime. Additionally, I love to crochet amigurumi, which are small and typically cute stuffed toys like dogs and cats, and larger stuffed toys.</p>
<div class="figure">
<img src="images/crochet_Ecoli.jpeg" alt="" />
<p class="caption">I’ve crocheted a small collection of <em>E. coli</em> amigurumi, and I hope to one day finally finish making one for each color of the rainbow, plus more. It’s quite soothing to just squish them and</p>
</div>
<p><img src="images/crochet_gray_cat.png" /></p>
<div class="figure">
<img src="images/crochet_cream_cat.png" alt="" />
<p class="caption">Among the many different cats and dogs I have crocheted, here are a couple of adorable chubby cats that I am making for a friend.</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
